devel/smake/Portfile

```diff
@@ -1,41 +1,96 @@
+checksums               rmd160  e154278ecbe7d778bc1d6766ed163c9963b1cc82 \
+                        sha256  a4270cdcca5dd69c0114079277b06e5efad260b0a099c9c09d31e16e99a23ff5 \
+                        size    5896292
 
-# https://trac.macports.org/ticket/31616
-compiler.blacklist-append llvm-gcc-4.2 {clang < 318}
-
-build.type		gnu
-build.args      CC=${configure.cc}
-use_parallel_build  no
-
-destroot.args       CC=${configure.cc}
-destroot.destdir	INS_BASE=${destroot}${prefix} \
-					MANDIR=man \
-					INSUSR=`id -u` INSGRP=`id -g`
-post-destroot	{
-	file delete -force ${destroot}${prefix}/share/man/man5 \
-		${destroot}${prefix}/lib ${destroot}${prefix}/include
+post-extract {
+                        copy ${filespath}/Gmake.smake ${worksrcpath}
 }
 
-livecheck.url       https://sourceforge.net/projects/s-make/files/
-livecheck.regex     ${name}-(\\d+(?:\\.\\d+)*)${extract.suffix}
+patchfiles              001-Makefile-Patch.diff
+
+post-patch {
+                        reinplace -locale C "s|/opt/schily|${prefix}|g" \
+                        ${worksrcpath}/DEFAULTS/Defaults.darwin \
+                        ${worksrcpath}/DEFAULTS/Defaults.mac-os10 \
+                        ${worksrcpath}/DEFAULTS_ENG/Defaults.darwin \
+                        ${worksrcpath}/DEFAULTS_ENG/Defaults.mac-os10 \
+                        ${worksrcpath}/libfind/find.c \
+                        ${worksrcpath}/libfind/find_main.c \
+                        ${worksrcpath}/librscg/scsi-remote.c \
+                        ${worksrcpath}/patch/tests/random/cmptest.sh \
+                        ${worksrcpath}/patch/tests/random/gentest.sh \
+                        ${worksrcpath}/smake/job.c \
+                        ${worksrcpath}/smake/make.c \
+                        ${worksrcpath}/smake/smake.1 \
+                        ${worksrcpath}/TEMPLATES/Defaults.gcc \
+                        ${worksrcpath}/TEMPLATES/Defaults.clang \
+                        ${worksrcpath}/TEMPLATES/Defaults.xcc
+}
+
+conflicts_build-append  cdrtools
+
+# https://trac.macports.org/ticket/31616
+compiler.blacklist      llvm-gcc-4.2 macports-llvm-gcc-4.2 {clang < 300}
+
+configure.ldflags-append -lintl
+
+universal_variant       no
+
+depends_build-append    port:gmake \
+                        port:gettext-runtime
+                        
+depends_lib-append      port:gettext-runtime
+
+build.type              gnu
+                        
+build.args-append       INS_BASE="${destroot}${prefix}" \
+                        INS_RBASE="${destroot}${prefix}" \
+                        DEFOSINCDIRS="${prefix}/include" \
+                        CC_OPT=${configure.optflags}
+
+destroot.destdir        INS_BASE="${destroot}${prefix}" \
+                        INS_RBASE="${destroot}${prefix}" \
+                        MANDIR=man \
+                        DEFINSUSR=${install.user} \
+                        DEFINSGRP=${install.group}
+
+use_parallel_build      no
+
```

Comment:
```suggestion
```

It successfully builds on my machine on parallel mode with `-j12`. Any reason to keep it?

Reply:
> @macsguy -- I was thinking of adding the following to the Portfile:
> 
> ```
> # NOTE:  Only the building of the bootstrapped version of smake will use parallel building.
> #        The final version of smake (and its static associated libraries) are built using
> #        the boostrapped version of smake which does NOT support parallel building.
> #        i.e.  the j flag is ignored by smake and the boostrapped version of smake.
> use_parallel_build      yes
> ```
> 
> Or would you prefer that I just delete all of the above?

The comment is definitely helpful for folks' understanding, so leave that.

As for `use_parallel_build yes`, that's the default, so no need for it. But given how this all works, you're welcome to include `use_parallel_build no`. Up to you.

Reply:
I think I will stick with `use_parallel_build no`.  The amount of compiling to make the bootstrapped version of `smake` is minimal.  So there is little speed improvement, if any, with parallel building enabled.

And beside with parallel building enabled, one will start seeing the following warnings:
`smake: Skipping illegal option '--jobserver-fds=5,6'.`

`--jobserver-fds` is only used with parallel building,

Reply:
@RobK88 it's clear. But I suggest to add a comment with explanation why it was disabled.